### DISCUSSION TITLE
Pinned apps shown instead of All apps

### DISCUSSION THREAD

**Discussion by kyrcy** [valinet/ExplorerPatcher#3529] [General]
With 22621.3880.66.4 on 24H2 (10.0.26100.1150) and Windows 11 start menu selected and Open Start in All apps by default ticked, only Pinned apps are shown. Why?

**Comment by pyrates999** [comment]
Not a beta version, sorry: https://support.microsoft.com/en-us/topic/july-9-2024-kb5040435-os-build-26100-1150-954078e3-6c6b-4b6d-885e-f5aa534524a6

Does this happen if you uninstall EP?

**Reply by Amrsatrio** [reply]
@pyrates999 This is an EP feature, showing "All apps" when the 11 start menu opens. I haven't tested if this feature still works on 24H2.

I recommend you to at least get a VM or a secondary device with Windows 11 installed so you can know the landscape of 11+EP better.

**Reply by kyrcy** [reply]
If this is an ExplorerPatcher feature that no longer works on 24H2, will it be fixed?
